[2024-12-15T22:11:04.034+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: covid_data_pipeline.validate_data manual__2024-12-15T22:05:57.227850+00:00 [queued]>
[2024-12-15T22:11:04.051+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: covid_data_pipeline.validate_data manual__2024-12-15T22:05:57.227850+00:00 [queued]>
[2024-12-15T22:11:04.052+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-12-15T22:11:04.077+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): validate_data> on 2024-12-15 22:05:57.227850+00:00
[2024-12-15T22:11:04.084+0000] {standard_task_runner.py:57} INFO - Started process 347 to run task
[2024-12-15T22:11:04.091+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'covid_data_pipeline', 'validate_data', 'manual__2024-12-15T22:05:57.227850+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/covid_pipeline_dag.py', '--cfg-path', '/tmp/tmpjvwo80dz']
[2024-12-15T22:11:04.097+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask validate_data
[2024-12-15T22:11:04.211+0000] {task_command.py:416} INFO - Running <TaskInstance: covid_data_pipeline.validate_data manual__2024-12-15T22:05:57.227850+00:00 [running]> on host efb61c2ab0bb
[2024-12-15T22:11:04.390+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ctw2012105@@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='covid_data_pipeline' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-15T22:05:57.227850+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-15T22:05:57.227850+00:00'
[2024-12-15T22:11:04.450+0000] {data_validator.py:34} INFO - Validating data schema
[2024-12-15T22:11:04.451+0000] {data_validator.py:50} INFO - Validating data quality
[2024-12-15T22:11:04.459+0000] {data_validator.py:82} INFO - All validations passed successfully
[2024-12-15T22:11:04.460+0000] {python.py:194} INFO - Done. Returned value was: Data validation completed
[2024-12-15T22:11:04.528+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=covid_data_pipeline, task_id=validate_data, execution_date=20241215T220557, start_date=20241215T221104, end_date=20241215T221104
[2024-12-15T22:11:04.601+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-12-15T22:11:04.673+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
